content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -76,9 +78,10 @@ consumed. They are distinct from
 [Services](/docs/concepts/services-networking/service/) are objects that can be read and modified
 through the Kubernetes API server.
 
-## Resource requests and limits of Pod and Container
+## Resource requests and limits of Pod and container
```

Comment:
```suggestion
## Resource requests and limits of Pods and containers
```

Reply:
If we change the wording we break existing hyperlinks (or we can explicitly set a custom fragment identifier)

Reply:
I think set the anchor manually and change the wording of the heading for grammatical reasons, but it's a nit